pubspec.yaml

```diff
@@ -12,7 +12,7 @@ platforms:
   macos:
 
 dependencies:
-  analyzer: ^6.0.0
+  analyzer: ^5.13.0
```

Comment:
why do you revert that?

Reply:
Thank you for your response. I attempted to upgrade to the latest version 6.0.0, but it doesn't work with this PR because of the incompatibility introduced by analyzer 6.0.0, which implemented 'final' and this affects the functionality. I tried to clone the analyzer project, but it's out of my reach. I hope you consider this regarding the PR. In my opinion, this PR will be closed due to being out of scope.

![image](https://github.com/bancolombia/dart-code-linter/assets/19521054/13c446cb-8f98-4cea-8700-77602fd16ccf)

Reply:
maybe you can complete issues in separated PR: feat: add job test on CI/CD pipeline maybe in one PR,
finally in other PR you can complete style: smell code.

Regards